detection_rules/rule_loader.py

```diff
@@ -164,7 +162,7 @@ def download_worker(pr_info):
         response = requests.get(rule_file.raw_url)
         try:
             raw_rule = pytoml.loads(response.text)
-            rule = Rule(rule_file.filename, raw_rule)
+            rule = TOMLRule(rule_file.filename, raw_rule)
             rule.gh_pr = pull
```

Comment:
This fails since `TOMLRule` is frozen

Reply:
Fixed in 5497331e660703d69b60509b62eebfc6d9404cb3
Made unfrozen, but kept the contents frozen